MSAL/src/public/configuration/MSALPublicClientApplicationConfig.h

```diff
@@ -95,6 +95,12 @@ to target MSAL at a specific test slice & flight. These apply to all requests ma
  */
 - (nonnull instancetype)initWithClientId:(NSString *)clientId;
 
+/**
+ Default is disable
```

Comment:
Can you add a little bit more detailed comment such as developer would understand why it is needed?

Reply:
Address, I will update in next commit.